File: app/src/main/java/hyunwook/co/kr/paging_livedata/entity/Word.java
package hyunwook.co.kr.paging_livedata.entity;

import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;
import android.support.annotation.NonNull;

@Entity(tableName = "word_table")
public class Word {

    @PrimaryKey
    @NonNull
    @ColumnInfo(name = "word")
    private String mWord;

    private Word(@NonNull String word) {
        this.mWord = word;
    }

    public String getWord() {
        return this.mWord;
    }
}

File: app/src/main/java/hyunwook/co/kr/paging_livedata/entity/Word.kt
package hyunwook.co.kr.paging_livedata.entity

import androidx.annotation.NonNull
import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "word_table")
class Word(@PrimaryKey @field:NonNull @ColumnInfo(name = "word") @NonNull val word: String)
